contracts/wallet.sol

```diff
@@ -250,7 +250,7 @@ library DailyLimitTrait {
         uint available;
         uint limitTimestamp;
         uint pending;
-        bool updateable;
+        bool controllerConfirmationRequired;
```

Comment:
I would personally go for a shorter name to be consistent like `protected` or `requireConfirmation` but this seems fine.

Reply:
I guess that this could be also 2FA required, which is done by controller but i guess that it is gonna be introduced in the next version.

Reply:
yeah, i think i wanted to make it explicit that it is `controller required` in the next version we can say `2FArequired` or similar after we have abstracted away the controller to a generic 2FA. Thanks guys